CondCore/SiPhase2TrackerPlugins/BuildFile.xml

```diff
@@ -0,0 +1,9 @@
+<use name="CondCore/ESSources"/>
+<use name="CondFormats/CondTest"/>
+<use name="CondFormats/DataRecord"/>
```

Comment:
the subsystem `CondFormats/CondTest/` doesn't exits:
see e.g.: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-26029/33554/buildClang.log
```
****WARNING: Invalid tool CondFormats/CondTest. Please fix src/CondCore/SiPhase2TrackerPlugins/BuildFile.xml file.
```

Reply:
Fixed